library/src/G4CMPCrystalGroup.cc

```diff
@@ -141,16 +140,56 @@ G4bool G4CMPCrystalGroup::FillOrthorhombic(G4double Cij[6][6]) const {
 
 G4bool G4CMPCrystalGroup::FillRhombohedral(G4double Cij[6][6]) const {
   G4double C11=Cij[0][0], C12=Cij[0][1], C13=Cij[0][2], C14=Cij[0][3];
-  G4double C15=Cij[0][4], C33=Cij[2][2], C44=Cij[3][3], C66=0.5*(C11-C12);
-
+  G4double  C33=Cij[2][2], C44=Cij[3][3], C66=0.5*(C11-C12);
+  
   Cij[1][1] = C11;	// Copy small number of individual elements
   Cij[1][2] = C13;
   Cij[1][3] = -C14;
-  Cij[1][4] = -C15;
-  Cij[3][5] = -C15;
+  Cij[1][4] = 0.0;
+  Cij[3][5] = 0.0;
   Cij[4][4] = C44;
   Cij[4][5] = C14;
 
+  //Filling the other terms of the matrix, one-by-one (IH)
+  Cij[2][1] = C13;
+  Cij[3][1] = -C14;
+  Cij[4][1] = -0.0;
+  Cij[5][3] = -0.0;
+  Cij[5][4] = C14;
+  Cij[5][5] = C66;
+  
+  Cij[3][0]=C14;
+  Cij[1][0]=C12;
+  Cij[2][0]=C13;
+  
+  ///////////Filling the other terms of the Matix with zeros (on top of those already initialized to zero and then never changed)
```

Comment:
Please don't use long strings of slashes, and please limit lines to 80 characters or less.